packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -96,22 +96,28 @@ const metadata: Meta<WaferMapArgs> = {
             WARNING - The wafermap is still in development and considered
             experimental. It is not recommended for application use.
         </div>
-        <nimble-wafer-map
-            colors-scale-mode="${x => x.colorScaleMode}"
-            ?die-labels-hidden="${x => x.dieLabelsHidden}"
-            die-labels-suffix="${x => x.dieLabelsSuffix}"
-            max-characters="${x => x.maxCharacters}"
-            orientation="${x => x.orientation}"
-            quadrant="${x => x.quadrant}"
-            :colorScale="${x => x.colorScale}"
-            :dies="${x => getDiesSet(x.dies, wafermapDieSets)}"
-            :highlightedValues="${x => getHighLightedValueSets(
+        <div class="wrapper">
```

Comment:
If we give the control a good default size then we don't need the wrapper anymore

Reply:
applied